src/AdminService.Validation/InstallAppRequestValidator.cs

```diff
@@ -57,10 +58,10 @@ public class InstallAppRequestValidator : AbstractValidator<InstallAppRequest>,
             .Cascade(CascadeMode.Stop)
             .NotEmpty().WithMessage("Login can't be empty.")
             .Must(x => char.IsLetter(x[0])).WithMessage("Login must start with a letter.")
-            .MinimumLength(3).WithMessage("Login is too short.")
+            .MinimumLength(6).WithMessage("Login is too short.")
             .MaximumLength(15).WithMessage("Login is too long.")
-            .Must(x => x.All(char.IsLetterOrDigit))
-            .WithMessage("Login must contain only letters or digits.");
+            .Must(x => LoginRegex.IsMatch(x))
+            .WithMessage("Login must contain only Latin letters or digits.");

```

Comment:
latin letters AND digits

Reply:
updates - login must contains only letters or letters and digits

Reply:
done